utils/cliutils/commandsflags.go

```diff
@@ -1380,6 +1381,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  workingDirs,
 		Usage: "[Optional] A comma-separated list of relative working directories, to determine audit targets locations.` `",
 	},
+	ExclusionsAudit: cli.StringFlag{
+		Name:  exclusions,
+		Usage: "[Default: *node_modules*;*target*;*venv*;*test*] Semicolon-separated list of exclusions. Exclusions can include the * and the ? wildcards.` `",
```

Comment:
Semicolon-separated list of exclusions. Exclusions can include the * and the ? wildcards.
-->
List of exclusions separated by semicolons, utilized to skip sub-projects from undergoing an audit. These exclusions may incorporate the * and ? wildcards.